### PR TITLE
fix(app subscription): change activation overlay

### PR DESCRIPTION
**Pull request by manojava-gk**
## Description

 activate button showing service activation screen instead of app activation

## Why

wrong overlay being shown on activation

## Issue

NA

## Checklist

Please delete options that are not relevant.

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally

### CODE REVIEW

**Comment by manojava-gk** [comment]
**DO NOT MERGE UNTILL NIDHI APPROVES**

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,8 @@
   - Enable search & filter
 - Serview Overview
   - Added Sub menu for active services in service overview
+- Bugfix - App Subscription
```
Comment (evegufy) [inline CHANGELOG.md:12]: Please but this fix under the already existing section for app subscription.

**Review by evegufy — CHANGES_REQUESTED** [review]

**Review by nidhigarg-bmw — APPROVED** [review]
lftm.

**Comment by evegufy** [comment]
@manojava-gk please update the dependencies file

**Comment by manojava-gk** [comment]
> @manojava-gk please update the dependencies file

there is no new dependency file in the summary. But this issue is happening for every PR now.

**Comment by evegufy** [comment]
IP issue created: https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/10499

**Comment by evegufy** [comment]
@manojava-gk you can rcommit the dependencies file now

**Review by evegufy — APPROVED** [review]

**Pull request merged**